admin/console/working-on-tickets.rst

```diff
@@ -13,19 +13,31 @@ check the EML-file itself.
 To get the first articles EML-file, you can use the following command.
 In our example the ticket number in question is ``101234``.
 
-.. code-block:: ruby
+.. code-block:: irb
 
-  >> Ticket.find_by(number:'101234').articles.first.as_raw.content
+   >> Ticket.find_by(number:'101234').articles.first.as_raw.content
 
 If needed, you can also get the raw content of later articles (you'll need to
 find the correct article though). Again, we expect ``101234`` to be our ticket
 number. In the first step we get all article IDs of the ticket, from the list
 we get, we can then get the articles content.
 
-.. code-block:: ruby
+Get article IDs:
+
+.. code-block:: irb
+
+   >> Ticket.find_by(number:'101234').article_ids
+
+Example output:
+
+.. code-block:: irb
+
```

Comment:
For this output code block, we can add `:class: no-copybutton` to its header, it will skip rendering the button.